howso/train_ts_ablation.amlg

```diff
@@ -0,0 +1,562 @@
+(null
+	#!DoesDataNeedGroupingById
+	(declare
+		(assoc
+			data []
+			id_indices [0]
+		)
+
+		(declare (assoc
+			one_id (= 1 (size id_indices))
+			id_index (first id_indices)
+			needs_grouping (false)
+		))
+
+		(if one_id
+			(let
+				(assoc
+					unique_ids_set
+						(if one_id
```

Comment:
one_id is already a truthy here

Reply:
it is only truthy by default, if user specifies multiple id_indices when calling this method, it'll be false.